ansible_base/jwt_consumer/common/auth.py

```diff
@@ -21,6 +24,11 @@
     "is_superuser",
     "is_system_auditor",
 ]
+# This setting allows a service to override which django cache we want to use
+cache_setting = getattr(settings, 'ANSIBLE_BASE_JWT_CACHE_NAME', 'default')
```

Comment:
This isn't used anywhere

Reply:
Good catch, the next line was supposed to use it.